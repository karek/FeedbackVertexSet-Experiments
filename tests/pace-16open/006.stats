471,503,51
