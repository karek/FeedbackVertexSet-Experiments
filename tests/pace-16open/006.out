11
317 111 176 0 9 462 46 191 131 63 231 
